package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "private": "true",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack --config webpack.config.js",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FrederickMih/RESTAURANT-PAGE.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FrederickMih/RESTAURANT-PAGE/issues"
+  },
+  "homepage": "https://github.com/FrederickMih/RESTAURANT-PAGE#readme",
+  "devDependencies": {
+    "bootstrap": "^5.0.1",
+    "node-sass": "^6.0.0",
+    "sass-loader": "^11.1.1",
+    "webpack": "^5.38.0",
+    "webpack-cli": "^4.7.0",
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.27.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.23.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.19.0"
+  },
+  "dependencies": {
+    "css-loader": "^5.2.6",
+    "style-loader": "^2.0.0"
```

Comment:
- [ ] All webpack related dependencies should be in devDependencies, these two dependencies are used by webpack when it is bundling you js files, and loading your css files and stylesheets. ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from the pathwright)